src/depends/common/Common.h

```diff
@@ -109,9 +109,9 @@ namespace dev
 
     // Map types.
     using StringMap = std::map<std::string, std::string>;
-    using BytesMap = std::map<bytes, bytes>;
+    using BytesMap = std::map<zbytes, zbytes>;
     using u256Map = std::map<u256, u256>;
-    using HexMap = std::map<bytes, bytes>;
+    using HexMap = std::map<zbytes, zbytes>;
```

Comment:
zHexMap?

Reply:
The idea is to distinguish names that have 'byte(s)' in them to zbyte(s) to stress this is Zilliqa's definition of byte and not std::byte. I'll skip this one as this isn't required.